foundation-config/src/main/java/io/servicecomb/config/ConfigurationSpringInitializer.java

```diff
@@ -43,4 +44,13 @@ protected Properties mergeProperties() throws IOException {
         }
         return properties;
     }
+
```

Comment:
the idea of DynamicProperties here is to replace direct use of archaius classes in our code, so that the testing and extending our framework to support other dynamic configurations, such as disconf, can be done without modification to our framework code.